cron.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+
+import json
+import re
+import requests
+import os
+from multiprocessing.pool import ThreadPool
+
+import openai
+import srt
+
+import numpy as np
+import pandas as pd
+
+from datetime import datetime, timedelta
+
+from langchain.embeddings import OpenAIEmbeddings
+from langchain.schema import Document
+from requests.exceptions import HTTPError
+from sklearn.cluster import KMeans
+
+DEV_ENV = False
+
+VISEXP = "https://storage.googleapis.com/data.gdeltproject.org/gdeltv3/iatv/visualexplorer"
+VICUNA = "http://fc6000.sf.archive.org:8000/v1"
+MODEL = "gpt-3.5-turbo"
+OUTPUT_FOLDER_NAME = "summaries"
+
+IDDTRE = re.compile(r"^.+_(\d{8}_\d{6})")
+
+CHANNELS = [
+  "ESPRESO",
+  "RUSSIA1",
+  "RUSSIA24",
+  "1TV",
+  "NTV",
+  "BELARUSTV",
+  "IRINN",
+]
+
+LM = "OpenAI" # language model
+CK = 30 # chunk size
+CT = 20 # cluster count
+DT = (datetime.now() - timedelta(hours=30)).date().strftime("%Y%m%d") # date
+LG = "English" # language
+
+THREAD_COUNT = 15
```

Comment:
For cron job, I can think of it being an even smaller number, maybe 10, because we want reliability over throughput.

Reply:
Done